docs/src/main/sphinx/security/ranger-access-control.md

```diff
@@ -0,0 +1,132 @@
+# Apache Ranger access control
+* Apache Ranger access control plugin supports use of Apache Ranger policies to authorize data access in Trino - like operations on catalogs, schemas, tables, columns.
+* Column-masking and row-filtering are supported by the plugin.
+* Accesses authorized by the plugin are audited for compliance purposes.
+
+## Requirements
+* Access to a Apache Ranger deployment having authorization policies to be enforced
+* Access to audit stores (Solr/HDFS/Log4J/S3) to save access audit logs
+* Apache Ranger 2.5.0 includes Trino service-def required by this access control plugin. In earlier versions of Apache Ranger, Trino service-def will have to be updated to the version [here](
+  https://github.com/apache/ranger/blob/ranger-2.5/agents-common/src/main/resources/service-defs/ranger-servicedef-trino.json).
+
+## Configuration
+Setup following configuration files to configure Apache Ranger as the authorizer in Trino:
+
+### /etc/trino/access-control.properties
+```properties
```

Comment:
Pull this apart into the actual config file separately from the description of the properties.

Reply:
You could maybe create a table with each property and its description.